src/ui/public/vislib/visualizations/gauge_chart.js

```diff
@@ -0,0 +1,71 @@
+import d3 from 'd3';
+import $ from 'jquery';
+import { VislibVisualizationsChartProvider } from './_chart';
+import { GaugeTypesProvider } from './gauges/gauge_types';
+
+export function GaugeChartProvider(Private) {
+
+  const Chart = Private(VislibVisualizationsChartProvider);
+  const gaugeTypes = Private(GaugeTypesProvider);
+
+  class GaugeChart extends Chart {
```

Comment:
Occasionally I'm seeing the gauge's render at different places in the screen. For example, Metric visualizations when choosing the Meter Gauge type render here:

<img width="1168" alt="screen shot 2017-05-18 at 11 07 34 am" src="https://cloud.githubusercontent.com/assets/627123/26209243/3ea28ce0-3bba-11e7-84c6-491822aecdf5.png">

And Circle Gauge Types render here:

<img width="1166" alt="screen shot 2017-05-18 at 11 07 42 am" src="https://cloud.githubusercontent.com/assets/627123/26209249/452ab93e-3bba-11e7-8193-7653e20714b1.png">